### DISCUSSION TITLE
How to update AGH on pi?

### DISCUSSION THREAD

**Discussion by BrickFlick** [AdguardTeam/AdGuardHome#4203] [Q&A]
I just installed AGH on my pi and I see this:
![image](https://user-images.githubusercontent.com/94026924/151022909-878f0753-9b52-4c9c-b016-0267822ec25c.png)

The link takes me to a list of files.  There are no instructions as to how to perform this update, and doing some research it seems many other people have the same question I do - *How* do we update?

I have read over these [thread1 ](https://www.reddit.com/r/Adguard/comments/pjb3cp/adguard_home_on_rpi_how_do_i_update/) and [thread2](https://www.reddit.com/r/Adguard/comments/pjb3cp/adguard_home_on_rpi_how_do_i_update/)
Resulting in:
![image](https://user-images.githubusercontent.com/94026924/151024872-3f5d66a2-a54d-4cfa-b3f0-3ddb12ebaa1f.png)

I (and many others) would greatly appreciate instructions on how to do this.  Thanks.

**Comment by ainar-g** [comment]
Hello and thanks for the report!

I've filed two separate issues about the problems here:

 *  #4208 
 *  #4209 

To answer your question directly, you need to:

1.  Backup your data.

2.  Download the new release archive and unpack it somewhere.

3.  Stop AGH.

4.  Replace the old binary with the new one from the archive.

5.  Restart AGH.
